tests/tmp/.gitignore

```diff
@@ -0,0 +1,2 @@
+/*
```

Comment:
Prosim, hod to o uroven vys. At tam nemame prazdny adresar.

https://github.com/contributte/console/blob/master/tests/.gitignore

Reply:
ok